### PR TITLE
JDBC-49 Properly serialize primitive types to string representation

### PR DESCRIPTION
**Pull request by rpastrana**
Signed-off-by: Pastrana, Rodrigo <rodrigo.pastrana@lexisnexis.com>

### CODE REVIEW

**Comment by rpastrana** [comment]
@garonsky please review

File: src/main/java/org/hpccsystems/jdbcdriver/HPCCPreparedStatement.java [hunk 1]
```diff
@@ -123,6 +134,60 @@ public ResultSet executeQuery() throws SQLException
...
+            else if (x instanceof java.sql.Date)
+                return ((java.sql.Date)x).toString();
+            else if (x instanceof Timestamp)
+                return ((Timestamp)x).toString();
+            else if (x instanceof InputStream)
+            {
+                ByteArrayOutputStream result = new ByteArrayOutputStream();
+                byte[] buffer = new byte[1024];
+                int length;
+                while ((length = ((InputStream)x).read(buffer)) != -1)
```
Comment (garonsky) [inline src/main/java/org/hpccsystems/jdbcdriver/HPCCPreparedStatement.java]: Can you just read until you read less than 1024 bytes?  It looks like you will call read again after reading less than 1024 bytes when there is nothing more to read.

File: src/main/java/org/hpccsystems/jdbcdriver/HPCCPreparedStatement.java [hunk 2]
```diff
@@ -123,6 +134,60 @@ public ResultSet executeQuery() throws SQLException
...
+                {
+                    result.write(buffer, 0, length);
+                }
+                return result.toString("UTF-8");
+
+                //Requires java 7
+                //try(java.util.Scanner s = new java.util.Scanner((InputStream)x, "UTF-8"))
+                //{
+                //    return s.useDelimiter("\\A").hasNext() ? s.next() : "";
+                //}
```
Comment (garonsky) [inline src/main/java/org/hpccsystems/jdbcdriver/HPCCPreparedStatement.java]: Remove commented out code?

Comment (rpastrana) [inline src/main/java/org/hpccsystems/jdbcdriver/HPCCPreparedStatement.java]: Keeping around for record.

**Comment by rpastrana** [comment]
@garonsky please review  		c850d09

**Pull request merged**